packages/api/app/Models/Technology.js

```diff
@@ -57,6 +57,20 @@ class Technology extends Model {
 				});
 		}
 
+		if (filters.keywords) {
+			const keywords = filters.keywords?.split(',') || [];
```

Comment:
Same here

Reply:
I will remove this block of code. This is no longer needed here